### DISCUSSION TITLE
无法使用TikTok

### DISCUSSION THREAD

**Discussion by Ccis996** [2dust/v2rayNG#3138] [General]
在提出问题前请先自行排除服务器端问题，同时也请通过搜索确认是否有人提出过相同问题。

### 预期行为
应该跟Clash一样能够使用TikTok

### 实际行为
Clash能够正常使用，这个却不可以。

### 复现方法
1.
2.
3.

### 日志信息
<details>

通过`adb logcat -s com.v2ray.ang GoLog V2rayConfigUtilGoLog Main`获取日志。请自行删减日志中可能出现的敏感信息。

如果问题可重现，建议先执行`adb logcat -c`清空系统日志再执行上述命令，再操作重现问题。
```
在这里粘贴日志
```
</details>

**Comment by fgr178707** [comment]
请尝试降级到1.8.19
https://github.com/2dust/v2rayNG/releases/tag/1.8.19

**Reply by V4444** [reply]
为啥呢？

**Comment by 582583241** [comment]
终于有人提这个问题了。我的也是如此！

**Comment by fgr178707** [comment]
> 终于有人提这个问题了。我的也是如此！

这边建议降级操作到1.8.19

**Comment by ShawnaStewart2000** [comment]
取消 Settings 里的 Enable Sniffing 就可以 Tiktok 了
可是，访问 Baidu 搜索 IP 得到的是真实IP
尝试了各种方法，国内部分域名使用的是本地IP

**Reply by Huayu528** [reply]
非常感谢，简体中文里就是把设置里启用流量探测的√去掉

**Reply by V4444** [reply]
试过了，不行。新版本关闭掉，还是不行

**Comment by V4444** [comment]
经过测试，的确是NG问题。使用相同节点，更换不同版本。1.8.19就能正常观看，后续的就不行。why?

**Comment by pipi-world** [comment]
路由设置里域名规则——在domain那里填geosite:tiktok     填好以后重新连接一下节点就可以了
